packages/frontend/src/components/LandingPage/LandingPageHero.tsx

```diff
@@ -96,7 +97,8 @@ export default ({}) => {
                 email: ''
             },
             setReceiptEmail: () => {},
-            onUpdate: onUpdate
+            onUpdate: onUpdate,
+            maxRankings: Number(process.env.REACT_APP_MAX_BALLOT_RANKS)
```

Comment:
I think it would be better to reference the environment variable in RankedBallotView

If someone leaves the max ranking undefined, and then we decide to update MAX_BALLOT_RANKS later then I want it to apply to all elections that left it undefined. RankedBallotView can look at the min between MAX_BALLOT_RANKS and ballotContext.maxRankings

I think undefined would be a good maxRankings value. 999999 would work as well, but I think undefined is more clear

Reply:
Ok, I updated it to reference it in RankedBallotView instead. Good catch